.changesets/feat_initial_delay_seconds_configuration_on_helm.md

```diff
@@ -0,0 +1,16 @@
+### Make initialDelaySeconds configurable for health probes in helm chart
+
+Currently `initialDelaySeconds` uses the default of `0`. This means that kubernetes will give router no time at all before it does the first probe. In practice this will always fail.
+
+This changes makes `initialDelaySeconds` configurable, but also makes its default value `1`.
+
+Can be configured as follows:
+
+```yaml
+router:
+  configuration:
+    health_check:
+      initialDelaySeconds: 5
```

Comment:
Not sure if its fine to extend `health_check` object here for k8s/helm purpuses.

Reply:
It's not. This wouldn't work without changes to support that configuration variable in the router. I don't think that's an appropriate fix, because the delay probably wouldn't be of benefit in the router, but only when the router is deployed within k8s (as you note).

I'd probably favour addressing this by adding something like this to values.yaml
```
probes:
  # -- Configure liveness probe
  liveness:
    initialDelaySeconds: 0
  # -- Configure readiness probe
  readiness:
    initialDelaySeconds: 0
```
We can then easily extend this if we want to add periodSeconds (or anything new) whenever.

I'd also prefer to leave the default at k8s default to reduce likelihood of "surprise".